common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -93,7 +94,8 @@ public void enableOverlays(Feature parent) {
     public void enableOverlay(Overlay enableOverlay) {
         if (!enableOverlay.shouldBeEnabled()) return;
 
-        this.enabledOverlays.add(enableOverlay);
+        enabledOverlays.add(enableOverlay);
+        WynntilsMod.registerEventListener(enableOverlay);
```

Comment:
This singular change opens up a whole lot of refactoring that can be done in features. Obviously I don't require you to do it, but if you want to :)

Reply:
Overall, I'm perfectly happy of leaving the overlays to you. :-) I was hoping I could trick you into writing this ContainerOverlay as well ;-) but I did not manage to express what I was going for. On the flip side, I learned a lot about the overlays in the process.

Reply:
I will clean them up, and review this, but after that I will have to do some projects of my own. User experience and scoreboard handler rewrite.